Chessington.GameEngine/Helpers.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Chessington.GameEngine
+{
+	public class Helpers
+    {
+        public List<Direction> AddMovesAndUpdateDirections(Board board, List<Direction> directions, Square currentSquare, int i, List<Square> moves, Player player)
```

Comment:
This line is quite long, please put each argument on a new line. Also could you give `i` a more descriptive name? It's not clear what it represents in this situation 🙂 

And this method can be static 🙂